templates/master.yaml

```diff
@@ -179,198 +205,31 @@ write_files:
       "metadata": {
         "name": "network-policy.net.alpha.kubernetes.io"
       },
-      "description": "fication for a network isolation policy",
+      "description": "Specification for a network isolation policy",
       "versions": [
         {
           "name": "v1alpha1"
         }
       ]
     }
-- path: /srv/fallback-server-dep.yml
```

Comment:
This wasn't actively removed by me, it just wasn't present in [the copy of cloudconfig I based my work on](https://github.com/kinvolk/aws-operator/blob/iaguis/aws/master-cloudconfig.yaml#L215).

So not every change in this PR has reasoning behind it.